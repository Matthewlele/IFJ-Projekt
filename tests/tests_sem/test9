if 1 == 1 then
  x = 1
  print x
else print "false"
  y = 2
end
