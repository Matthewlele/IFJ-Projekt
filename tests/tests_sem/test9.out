expected SYNTAX ERR: line 4
